code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -1273,11 +1273,11 @@ nobiliumsuppression = INFINITY
 	if(cached_gases[/datum/gas/bz][MOLES] - consumed_amount < 0)
 		return NO_REACTION
 	if(cached_gases[/datum/gas/bz][MOLES] < 30)
-		radiation_pulse(location, consumed_amount, 1, TRUE, FALSE)
+		radiation_pulse(location, consumed_amount * 20, 1, TRUE, FALSE)
```

Comment:
"""""tiny"""""

Reply:
is tiny tho, because the maximum amount of radiation is 600 rads, all non contaminant and a range 1 pulse. before was 30 an amount so tiny that it was barely noticeable. To confront, normal SM radiation are about 1500 rads

Reply:
if it would make you feel better i could rename it to reasonable buff

Reply:
Nah, it's just me being pedantic. You don't usually think of a 20x buff as tiny so this is fun to see.